model/src/main/java/com/redhat/ipaas/model/integration/Integration.java

```diff
@@ -31,9 +31,7 @@
 @JsonDeserialize(builder = Integration.Builder.class)
 public interface Integration extends WithId<Integration>, WithName, Serializable {
 
-    public static enum Type {Activated, Deactivated, Draft};
-
-    public static enum Phase {Pending, Running, Succeeded, Failed, Unknown};
+    public static enum Status { Draft, Activated, Deactivated, Pending, Deleted};
```

Comment:
Removing the Pending status.  current state should be set to null where Pending was being used before.